opensanctions/crawlers/us_congress.py

```diff
@@ -0,0 +1,122 @@
+from functools import cache
+import json
+import os
+from typing import Optional
+from urllib.parse import urlencode, urljoin
+from datetime import timedelta
+
+from opensanctions import helpers as h
+from opensanctions.core import Context
+from zavod.entity import Entity
+from zavod.parse.positions import make_position
+from zavod import settings
+
+
+API_KEY = os.environ.get("OPENSANCTIONS_US_CONGRESS_KEY")
+LIMIT = 250
+CACHE_DAYS = 5
+AFTER_OFFICE = 5 * 365
+
+
+# TODO: Copied from wd_pep, factor out
+@cache
+def to_date(days: int) -> str:
+    dt = settings.RUN_TIME - timedelta(days=days)
+    return dt.isoformat()[:10]
+
+
+def make_occupancy(
+    context: Context,
+    person,
+    position,
+    start_date,
+    end_date: Optional[str],
+    no_end_implies_current: bool,
+) -> Optional[Entity]:
+    """Helper for making occupancies if they meet our criteria for PEP position occupancy"""
+    # TODO: Change dates to be dates to generalise
+    # TODO: The general version will need more of the logic from wd_peps than just this clause
+    if end_date is None or end_date > to_date(AFTER_OFFICE):
+        occupancy = context.make("Occupancy")
+        parts = [person.id, position.id, start_date, end_date]
+        occupancy.id = context.make_id(*parts)
+        occupancy.add("holder", person)
+        occupancy.add("post", position)
+        occupancy.add("startDate", start_date)
+        occupancy.add("endDate", end_date)
+        if no_end_implies_current and not end_date:
+            status = "Current"
+        else:
+            status = "Ended"
```

Comment:
for this source this is fine, but the logic should actually be that it's only ended if there's an end date, and anything else is Unknown or Unsure or whatever we settle on